<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="picture_cam1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>512</width>
     <height>384</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="picture_cam2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>512</width>
     <height>384</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera 2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>720</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>720</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>720</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>30</y>
     <width>593</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item row="10" column="5">
     <widget class="QLabel" name="label_36">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="label_34">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item row="10" column="6">
     <widget class="QSpinBox" name="tresh_2_V_upper"/>
    </item>
    <item row="10" column="4">
     <widget class="QSpinBox" name="tresh_2_S_upper"/>
    </item>
    <item row="7" column="6">
     <widget class="QSpinBox" name="tresh_1_V_upper">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="14" column="3">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="14" column="2">
     <widget class="QSpinBox" name="cam2_roi_y_2">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="15" column="2">
     <widget class="QSpinBox" name="cam2_roi_y_3">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="15" column="3">
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="6">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="13" column="6">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="8">
     <widget class="QSpinBox" name="cam1_roi_y_11">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="3" column="6">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="13" column="2">
     <widget class="QSpinBox" name="cam2_roi_y_1">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="13" column="8">
     <widget class="QSpinBox" name="cam2_roi_y_9">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="14" column="8">
     <widget class="QSpinBox" name="cam2_roi_y_10">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="15" column="8">
     <widget class="QSpinBox" name="cam2_roi_y_11">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="13" column="5">
     <widget class="QSpinBox" name="cam2_roi_y_5">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="15" column="6">
     <widget class="QLabel" name="label_28">
      <property name="text">
       <string>11</string>
      </property>
     </widget>
    </item>
    <item row="14" column="6">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="14" column="5">
     <widget class="QSpinBox" name="cam2_roi_y_6">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="15" column="5">
     <widget class="QSpinBox" name="cam2_roi_y_7">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_29">
      <property name="text">
       <string>Color Threshold 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="8">
     <widget class="QSpinBox" name="cam1_roi_y_10">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Rois Cam2</string>
      </property>
     </widget>
    </item>
    <item row="10" column="3">
     <widget class="QLabel" name="label_35">
      <property name="text">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Rois Cam 1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QSpinBox" name="cam1_roi_x_10">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QSpinBox" name="cam1_roi_x_9">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="4" column="7">
     <widget class="QSpinBox" name="cam1_roi_x_11">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="label_47">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label_48">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label_46">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QLabel" name="label_49">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QSpinBox" name="cam1_roi_y_9">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QSpinBox" name="cam1_roi_y_8">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="15" column="4">
     <widget class="QSpinBox" name="cam2_roi_x_7">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <widget class="QSpinBox" name="cam2_roi_x_5">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QSpinBox" name="cam2_roi_x_6">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="12" column="2">
     <widget class="QLabel" name="label_51">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label_50">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="15" column="7">
     <widget class="QSpinBox" name="cam2_roi_x_11">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="14" column="7">
     <widget class="QSpinBox" name="cam2_roi_x_10">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="13" column="7">
     <widget class="QSpinBox" name="cam2_roi_x_9">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="12" column="5">
     <widget class="QLabel" name="label_52">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="12" column="7">
     <widget class="QLabel" name="label_54">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3">
     <widget class="QLabel" name="label_31">
      <property name="text">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item row="12" column="8">
     <widget class="QLabel" name="label_55">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="12" column="4">
     <widget class="QLabel" name="label_53">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="label_32">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QSpinBox" name="tresh_1_H_upper">
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="label_30">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4">
     <widget class="QSpinBox" name="tresh_1_S_upper">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QSpinBox" name="cam2_roi_x_3">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QSpinBox" name="cam2_roi_x_1">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QSpinBox" name="cam2_roi_x_2">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="16" column="2">
     <widget class="QSpinBox" name="cam2_roi_y_4">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QSpinBox" name="cam2_roi_x_4">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QLabel" name="label_38">
      <property name="text">
       <string>Middle Point Cam1</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QLabel" name="label_37">
      <property name="text">
       <string>Offsets</string>
      </property>
     </widget>
    </item>
    <item row="18" column="1">
     <widget class="QSpinBox" name="cam1_offset_middle_point">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="19" column="4">
     <widget class="QSpinBox" name="cam1_offset_4">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QLabel" name="label_40">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QLabel" name="label_42">
      <property name="text">
       <string>Middle Point Cam2</string>
      </property>
     </widget>
    </item>
    <item row="19" column="3">
     <widget class="QSpinBox" name="cam1_offset_3">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="19" column="2">
     <widget class="QSpinBox" name="cam1_offset_2">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="22" column="3">
     <widget class="QSpinBox" name="cam2_offset_3">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="22" column="1">
     <widget class="QSpinBox" name="cam2_offset_1">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="16" column="4">
     <widget class="QSpinBox" name="cam2_roi_x_8">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QLabel" name="label_39">
      <property name="text">
       <string>Offsets Cam1</string>
      </property>
     </widget>
    </item>
    <item row="16" column="5">
     <widget class="QSpinBox" name="cam2_roi_y_8">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="21" column="1">
     <widget class="QSpinBox" name="cam2_offset_middle_point">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="22" column="4">
     <widget class="QSpinBox" name="cam2_offset_4">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <widget class="QLabel" name="label_41">
      <property name="text">
       <string>Offset Cam2</string>
      </property>
     </widget>
    </item>
    <item row="19" column="1">
     <widget class="QSpinBox" name="cam1_offset_1">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="label_43">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item row="22" column="2">
     <widget class="QSpinBox" name="cam2_offset_2">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_56">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Upper</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QSpinBox" name="tresh_1_H_lower">
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="cam1_roi_x_2">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_44">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QSpinBox" name="cam1_roi_x_5">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QSpinBox" name="cam1_roi_x_4">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QSpinBox" name="cam1_roi_x_3">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_45">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item row="8" column="6">
     <widget class="QSpinBox" name="tresh_1_V_lower">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QSpinBox" name="tresh_1_S_lower">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QSpinBox" name="cam1_roi_y_4">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QSpinBox" name="cam1_roi_x_8">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QSpinBox" name="cam1_roi_x_6">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QSpinBox" name="cam1_roi_x_7">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QSpinBox" name="cam1_roi_y_2">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="cam1_roi_x_1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_33">
      <property name="text">
       <string>Color Threshold 2</string>
      </property>
     </widget>
    </item>
    <item row="10" column="2">
     <widget class="QSpinBox" name="tresh_2_H_upper"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QSpinBox" name="cam1_roi_y_7">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QSpinBox" name="cam1_roi_y_5">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QLabel" name="label_57">
      <property name="text">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_59">
      <property name="text">
       <string>Lower</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QSpinBox" name="cam1_roi_y_6">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QSpinBox" name="cam1_roi_y_3">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QLabel" name="label_58">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="cam1_roi_y_1">
      <property name="maximum">
       <number>1280</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="label_60">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_61">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Upper</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_62">
      <property name="text">
       <string>Lower</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="3">
     <widget class="QLabel" name="label_63">
      <property name="text">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QLabel" name="label_64">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item row="11" column="2">
     <widget class="QSpinBox" name="tresh_2_H_lower"/>
    </item>
    <item row="11" column="4">
     <widget class="QSpinBox" name="tresh_2_S_lower"/>
    </item>
    <item row="11" column="6">
     <widget class="QSpinBox" name="tresh_2_V_lower"/>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Thresh</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QSpinBox" name="tresh_2"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Thresh</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QSpinBox" name="tresh_1"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="openButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>720</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
